Cleaned up the Modal styling

VincentPheng opened the pull request:
Added functionality of modal to update both the meals and clients database
Changed Frozen day selection from radio to drop down
Decomposed Modal component

EmilyGavrilenko committed:
Pull changes

EmilyGavrilenko committed:
Merge branch 'master' into vincent

VincentPheng committed:
Cleaned up the Modal styling
Added functionality of modal to update both the meals and clients database
Changed Frozen day selection from radio to drop down
Decomposed Modal component

EmilyGavrilenko reviewed (changes requested):
Can you please update the ClientTableContainer to display the Client Modal, similar to how you display the Route Modal on the Route Homepage.

EmilyGavrilenko committed:
merge

VincentPheng committed:
Updated Client Page Modal to reflect the route table modal
Added route number editablity for client modal
Added white bag row and updated respective backend method

VincentPheng committed:
Merge remote-tracking branch 'origin/master' into vincent

VincentPheng committed:
Added proper database flags to avoid unnecessary backend calls

EmilyGavrilenko committed:
Merge branch 'vincent' of https://github.com/hack4impact-calpoly/meals-that-connect into vincent

EmilyGavrilenko committed:
Rename # of meals to Brown Bag

EmilyGavrilenko reviewed (approved):
Looks good!
